.env-cmdrc

```diff
@@ -59,5 +59,10 @@
   "pdf-invoices": {
     "CX_BASE_URL": "https://api.cg79x9wuu9-eccommerc1-s5-public.model-t.myhybris.cloud",
     "CX_PDF_INVOICES": "true"
+  },
+  "myaccount-v2":{
+    "CX_BASE_URL": "https://api.cg79x9wuu9-eccommerc1-s5-public.model-t.myhybris.cloud",
+    "CX_PDF_INVOICES": "true",
+    "CX_MYACCOUNT_V2": "true"
```

Comment:
[MINOR]
For consistency, let's treat "my account" as 2 words:
- `my-account-v2`
- `CX-MY_ACCOUNT_V2`

It's not our public API, that's why level is [MINOR], not [MEDUM]. Can be fixed anytime, even after release